build.sbt

```diff
@@ -649,11 +649,13 @@ lazy val root = (project in file("."))
     publish := {},
     publishLocal := {},
     commands ++= ScriptCommands.all,
-    // Generate (Product|TupleN|Function|AbstractFunction)*.scala files. Once the ANT build is gone,
-    // we should move them into a managedSources dir instead of overwriting sources checked into git.
-    commands += Command.command("genprod") { state =>
-      val dir = (((baseDirectory in ThisBuild).value) / "src" / "library" / "scala").getPath
-      genprod.main(Array(dir))
+    // Generate (Product|TupleN|Function|AbstractFunction)*.scala files and scaladoc stubs for all
+    // AnyVal sources. Once the ANT build is gone, we should move them into a managedSources dir instead
+    // of overwriting sources checked into git.
```

Comment:
Not checking files into git, although desirable in theory, would be problematic for the source maps of Scala.js.